projects/mercury/src/metadata/views/MetadataView.js

```diff
@@ -141,59 +141,69 @@ export const MetadataView = (props: MetadataViewProperties) => {
+    const renderFacets = (view) => (
         <Grid container item direction="column" justifyContent="flex-start" spacing={1}>
-            {facetsEx.map(facet => {
-                const facetOptions = ofRangeValueType(facet.type) ? [facet.min, facet.max] : facet.values;
-                const activeFilter = [...filterCandidates, ...filters].find(filter => filter.field === facet.name);
-                let activeFilterValues = [];
-                if (activeFilter) {
-                    activeFilterValues = ofRangeValueType(facet.type) ? [activeFilter.min, activeFilter.max] : activeFilter.values;
-                }
-                return facetOptions && facetOptions.length > 0 && (
-                    <Grid key={facet.name} item>
-                        <Facet
-                            type={facet.type}
-                            title={facet.title}
-                            options={facetOptions}
-                            onChange={(values) => updateFilterCandidates(facet, values)}
-                            extraClasses={classes.facet}
-                            activeFilterValues={activeFilterValues}
-                            clearFilter={() => handleClearFilter(facet.name)}
-                        />
-                    </Grid>
-                );
-            })}
-            <Grid
-                container
-                spacing={1}
-                className={`${classes.confirmFiltersButtonBlock} ${filterCandidates.length > 0 && classes.confirmFiltersButtonBlockActive}`}
-            >
-                <Grid item xs={4}>
-                    <Button
-                        onClick={clearFilterCandidates}
-                        variant="contained"
-                        color="default"
-                        className={classes.confirmFiltersButton}
-                        disabled={filterCandidates.length === 0}
-                    >
-                        Cancel
-                    </Button>
-                </Grid>
-                <Grid item xs={8}>
-                    <Button
-                        onClick={applyFilters}
-                        variant="contained"
-                        color="secondary"
-                        className={classes.confirmFiltersButton}
-                        disabled={filterCandidates.length === 0}
-                    >
-                        Apply filters
-                    </Button>
-                </Grid>
-            </Grid>
+            <div className={classes.facetHeaders} style={{textTransform: 'uppercase'}}>{view.title}</div>
+            {facetsEx.map(facet => (facet.name.toLowerCase().startsWith(view.name.toLowerCase()) ? renderSingleFacet(facet) : ""))}
+            {
+                // location is the collection location, which we will group under resources
+                (view.name.toLowerCase() === 'resource') ? facetsEx.map(facet => (facet.name.toLowerCase().startsWith('location') ? renderSingleFacet(facet) : "")) : ""
```

Comment:
Same here.

Reply:
You can also directly use `collectionsFacet` here, which is already defined separately.